frontend/js/src/stats/UserEntityChart.tsx

```diff
@@ -239,7 +244,7 @@ export default class UserEntityChart extends React.Component<
             count: elem.listen_count,
             caaID: elem.caa_id,
             caaReleaseMBID: elem.caa_release_mbid,
-            artists: elem.artists
+            artists: elem.artists,
```

Comment:
this change was made by prettier, looks like there are a few outstanding lint issues? the dev environment automatically runs prettier, so there's no good way to get around including this diff in the PR